### ISSUE TITLE
Ephemeral workspace is failing to start due to "the node was low on resource ephemeral-storage"

### ISSUE THREAD

**Issue by ibuziuk** [redhat-developer/rh-che#1372]
So, far was able to repro only once with che 7 ephemeral workspace:
![ephemeral](https://user-images.githubusercontent.com/1461122/56230975-a233e800-607d-11e9-8012-3076b34cc1ea.png)

**Comment by ibuziuk** [comment]
@l0rd @pbergene wdyt?

**Comment by pbergene** [comment]
Ephemeral storage isn't an infinite resource, so would have to scope out requirements vs what is available etc.

**Comment by ibuziuk** [comment]
@pbergene and what is the current state? how much ephemeral storage is available per node?

**Comment by Katka92** [comment]
This issue is monitored by our periodic tests and happens almost every day. The test is running Vert.x and Maven stack, the issue is reproducible for both of them. From time to time workspace failed on starting, but more often workspace is switched to offline while running a maven build or starting/testing LS server. In the events, we can see that the reason is low ephemeral storage.

**Comment by Katka92** [comment]
Today we've seen another issue than can probably be related to Evicted. The maven build and run tasks failed with `Terminal failed to connect. See more in 'Output'.`. There is an `evicted` event in events report. 

[1] events: http://artifacts.ci.centos.org/devtools/rhche/devtools-rh-che-periodic-prod-2/1956/events_report.txt
[2] Screenshot from the test: http://artifacts.ci.centos.org/devtools/rhche/devtools-rh-che-periodic-prod-2/1956/e2e_report/report/RhChe_E2E_Java_Maven_test_Validation_of_workspace_build_and_run_Build_application/screenshot-Build_application.png

**Comment by ScrewTSW** [comment]
Seen today twice at production cluster 2 x2
https://ci.centos.org/job/devtools-rh-che-periodic-prod-2/2072/console
https://ci.centos.org/job/devtools-rh-che-periodic-prod-2/2073/console

**Comment by ScrewTSW** [comment]
Seen today twice at production cluster 2 x2
https://ci.centos.org/job/devtools-rh-che-periodic-prod-2/2072/console
https://ci.centos.org/job/devtools-rh-che-periodic-prod-2/2073/console
and on production cluster 2a x1
https://ci.centos.org/view/Devtools/job/devtools-rh-che-periodic-prod-2aProd/1861/consoleFull

**Comment by ScrewTSW** [comment]
Seen again on production cluster 1b x1
https://ci.centos.org/view/Devtools/job/devtools-rh-che-periodic-prod-1b/1873/console

**Comment by Katka92** [comment]
We are still observing this issue.

**Comment by ibuziuk** [comment]
Closing in favour of https://github.com/redhat-developer/rh-che/issues/1770
